src/SymbolTable.cpp

```diff
@@ -148,14 +148,27 @@ Symbol* SymbolTable::getAddressSymbol(int addr, MemoryLayout* ml)
 Symbol* SymbolTable::getAddressSymbol(const QString& label, bool case_sensitive)
 {
 	for (auto it = addressSymbols.begin(); it != addressSymbols.end(); ++it) {
-		if (it.value()->text().compare(label, Qt::CaseSensitive)==0)
+		if (it.value()->text().compare(label, Qt::CaseSensitive) == 0)
 			return it.value();
-		if (!case_sensitive && it.value()->text().compare(label, Qt::CaseInsensitive)==0)
+		if (!case_sensitive && it.value()->text().compare(label, Qt::CaseInsensitive) == 0)
 			return it.value();
 	}
 	return nullptr;
 }
 
+[[nodiscard]] QStringList SymbolTable::findAddressSymbolsByAddress(int addr, MemoryLayout* ml)
```

Comment:
(No need to change for this PR.)

A more general (more reusable) method would return a collection of `Symbol*` instead of only extracting the `text` field of the matching symbols.